robot-server-plugin/src/main/kotlin/com/intellij/remoterobot/services/IdeRobot.kt

```diff
@@ -236,6 +237,27 @@ class IdeRobot {
         }
     }
 
+    fun makeScreenshot(): Result<ByteArray> {
+        return getResult(RobotContext(robot)) { ctx ->
+            robot.makeScreenshot()
+        }
+    }
+
+    fun makeScreenshot(componentId: String): Result<ByteArray> {
+        val componentContext =
+            componentContextCache[componentId] ?: throw IllegalStateException("Unknown component id $componentId")
+        return getResult(RobotContext(robot)) { ctx ->
```

Comment:
Why can't we use `componentContext` here instead of making a new`robotContext`

Reply:
fixed